SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Reporting security issues
+
+If you think you have found a security vulnerability, please send a report to security@datahubproject.io. This address can be used for all of Acryl Data’s open source and commercial products (including but not limited to DataHub and Acryl Data). We can accept only vulnerability reports at this address.
+
+Please encrypt your message to us; please use our PGP key. The key fingerprint is:
```

Comment:
We probably want PGP encryption to be optional and not required - looks like that's what kubernetes does

Reply:
ah ok! I'll go ahead and make that change - I'll just open a new PR for datahub-project/datahub